includes/Dashboard/Templates/Settings.php

```diff
@@ -634,4 +758,39 @@ function get_dokan_categories() {
 
         return apply_filters( 'dokan_category', $dokan_category );
     }
+
+    /**
+     * Get proper heading for payments of vendor dashboard payment settings
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @param string $slug
+     * @param string $heading
+     *
+     * @return string
+     */
+    private function get_payment_heading( $slug, $heading ) {
+        switch ( $slug ) {
+            case 'bank':
+            case 'bank/edit':
+                $heading = __( 'Add Bank Account', 'dokan-lite' );
+                break;
+
+            case 'paypal':
+            case 'paypal/edit':
+                $heading = __( 'Paypal Settings', 'dokan-lite' );
+                break;
+        }
+
+        /**
+         * To allow new payment extension give their own heading
+         *
+         * @since DOKAN_LITE_SINCE
+         *
+         * @param string $heading previous heading
+         */
+        $heading = apply_filters( "dokan_vendor_dashboard_payment_settings_heading", $heading, $slug );
```

Comment:
It might be better if the filter would be changed to something like:
`dokan_withdraw_method_settings_title`